### ISSUE TITLE
Datepicker thorugh error while set default tomorrow date and Today is min date

### ISSUE THREAD

**Issue by ahirhasmukh** [Promact/md2#116]
Datepicker thorugh error while set default tomorrow's date and Today's minimum date but some time error occurred. so please review my error and suggest me where I was wrong.

**.html code**

`<md2-datepicker type="date" formControlName="specialDate" [min]="minStartDate" placeholder="Date" class="bottom-date-picker"></md2-datepicker>`

**.ts code** 

```
date: string = moment().add(1, 'days').format( 'DD MMM, YYYY');
getFullDate: Date = new Date();
minStartDate: Date;
this.minStartDate = new Date(this.getFullDate.setDate(this.getFullDate.getDate()));

```
**Please check below screenshot.**

![datepicker](https://cloud.githubusercontent.com/assets/8941195/23496446/46b20594-ff44-11e6-84bd-6f9aec959041.png)

![console](https://cloud.githubusercontent.com/assets/8941195/23496458/54dc03c2-ff44-11e6-980b-a1fdba9445d1.png)

**Comment by dharmeshpipariya-zz** [comment]
@smartHasmukh we have updated date issue you can use it in next release

**Comment by dharmeshpipariya-zz** [comment]
We have fixed this issue in release @0.0.17
